src/api/entities/Proposal/index.ts

```diff
@@ -1,13 +1,11 @@
-import BigNumber from 'bignumber.js';
-
 import { Entity } from '~/base';
 import { Context } from '~/context';
 
 /**
  * Properties that uniquely identify a Proposal
  */
 export interface UniqueIdentifiers {
-  pipId: BigNumber;
+  pipId: number;
```

Comment:
This should still be a `BigNumber` to be consistent with other entities such as authorization